cli/cmd/package_manifest.go

```diff
@@ -252,6 +263,45 @@ func (c *cliState) detectActiveKernel() (string, bool) {
 	return strings.TrimSuffix(string(kernel), "\n"), true
 }
 
+func (c *cliState) IsEsmEnabled() bool {
+	type uaStatusFile struct {
+		SchemaVersion string `json:"_schema_version,omitempty"`
+		Status        string `json:"execution_status,omitempty"`
+		Services      []struct {
+			Name   string `json:"name,omitempty"`
+			Status string `json:"status,omitempty"`
+		} `json:"services,omitempty"`
+	}
+
+	if file.FileExists(procUAStatusFile) {
+		c.Log.Debugw("detecting ubuntu ESM support", "file", procUAStatusFile)
+		uaStatusBytes, err := ioutil.ReadFile(procUAStatusFile)
+		if err != nil {
+			c.Log.Warnw("unable to read UA status file", "error", err)
+			return false
```

Comment:
Do we want to return false here and not error?

Reply:
This if fine for now since we don't want to stop a scan if we can't detect if this feature is enabled or not.